app/views/provider_interface/organisation_settings/show.html.erb

```diff
@@ -4,16 +4,41 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l"><%= t('page_titles.provider.organisation_settings') %></h1>
     <ul class="govuk-list govuk-list--spaced">
```

Comment:
The prototype has one `ul` per provider. The spacing also looks a bit off in the review app – might be fixed by changing this

Reply:
Good spot, I've modified it so that each provider has it's own `ul`

Reply:
Looks good now 👍